addons/web/static/src/polyfills/array.js

```diff
@@ -0,0 +1,8 @@
+if (!Array.prototype.at) {
+    Array.prototype.at = function (index) {
+        if (index >= 0) {
+            return this[index];
+        }
+        return this[this.length + index];
```

Comment:
worth noting that this is not compliant with the spec as this will access properties that are out of bounds which has implications among other things with proxies (and also for weird use cases with properties at negative indices greater than the length), which may impact reactivity. I don't think it's much of a problem but worth keeping in mind.